src/components/pages/roster-page/roster-page.tsx

```diff
@@ -147,9 +72,33 @@ export const RosterPage = () => {
         onClose: () => { setAlertDialogProps({ open: false }); },
       });
     }
-  }
+  }, [orgId]);
+
+  const initializeTable = useCallback(async () => {
+    dispatch(AppFrame.setPageLoading(true));
+    await initializeRosterColumnInfo();
+    await reloadTable();
+    dispatch(AppFrame.setPageLoading(false));
+  }, [dispatch, initializeRosterColumnInfo, reloadTable]);
+
+  useEffect(() => {
+    initializeTable().then();
+  }, [initializeTable]);
+
+  //
+  // Functions
+  //
 
-  function handleFileInputChange(e: ChangeEvent<HTMLInputElement>) {
+  const handleChangePage = (event: MouseEvent<HTMLButtonElement> | null, pageNew: number) => {
+    setPage(pageNew);
+  };
+
+  const handleChangeRowsPerPage = (event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+    setRowsPerPage(parseInt(event.target.value));
+    setPage(0);
```

Comment:
Can we calculate the page based on which page the first row of the table is going to be on?  For example, if they were on page 6 showing 10 rows per page, they should be on page 2 if they switch to 50 rows per page, or page 3 if they switch to 25 rows per page.

Reply:
Yeah I think that would be better UX for sure.